docs/build/quickstarts.md

```diff
@@ -1,3 +1,200 @@
-# Quickstarts
+# Blockchain-based deployments quickstart
 
-Coming soon.
+Fluence's mode of a decentralized cloud allows to deploy WebAssembly functions and pay with tokens for their execution. Payments go to Providers, who host these functions.
+
+## What are deals
+
+Deal represents a request from Developer to host a set of WebAssembly functions for a specified price. Providers are able to join these deals in order to host functions and receive tokens for that.
+
+In essense, Deal specifies what to deploy, and how much to pay for that.
```

Comment:
```suggestion
In essence, Deal specifies what to deploy, and how much to pay for that.
```

Reply:
how about:

Deals originate as a request from a developer to have a set of Webassembly modules, aka services, hosted with the purpose of execution for a stipulated price. Capacity providers finding the service type and proposed payment acceptable can join the deal and in doing so fill available deal capacity. 
A critical non-monetary aspect of a deal is the immutable representation of a set of Wasm modules’ IPFS content identifiers (CID) `AppCID` that points to  the `Worker Definition` data structure stored on IPFS. Thus, providers participating in a deal can easily locate, retrieve and verify a deal’s  service “package.”